pom.xml

```diff
@@ -296,6 +296,10 @@
 		<logback.version>1.1.11</logback.version>
 		<httpclient.version>4.5.12</httpclient.version>
 		<httpcore.version>4.4.13</httpcore.version>
+		<commonscodec.version>1.13</commonscodec.version>
+		<commonsio.version>2.7</commonsio.version>
+		<commonslang3.version>3.11</commonslang3.version>
+		<commonstext.version>1.9</commonstext.version>
```

Comment:
i don't object, exactly, but why create properties for these if they're only referenced once, in the same file?

Reply:
Hmm yeah, I went a little overboard here... 

Personally, I find it a tiny bit easier to spot the versions (instead of finding "version")... 
Though the downside is that the IDE cannot be used anymore to check for newer versions using delete-and-autocomplete-again... so I'll just revert this... less lines = less bugs